aether-ui/aether/ui/assets/apps/pipeline/Pipeline.jsx

```diff
@@ -324,14 +333,12 @@ class Pipeline extends Component {
         key={newContract.id}
         className={`pipeline-tab ${newContract.id === this.props.contract.id ? 'active' : ''}`}
       >
-        <span className='contract-name'>{ newContract.name }</span>
-        <span className='status white' />
-
-        <div
-          className={`btn-icon settings-button ${this.state.showSettings ? 'active' : ''}`}
-          onClick={() => { this.props.selectSection(CONTRACT_SECTION_SETTINGS) }}>
-          <i className='fas fa-ellipsis-h' />
-        </div>
+        <span className='contract-name new'>
+          <FormattedMessage
+            id='pipeline.tab.newContract'
+            defaultMessage='new Contract'
```

Comment:
The actual name generated for the new contract is: 
```suggestion
            defaultMessage={newContract.name}
```

Reply:
I changed that on purpose, because I wanted the tab to say "new Contract" when at the same time keeping the suggested new contract name for the text input.